state-sync/src/coordinator.rs

```diff
@@ -269,31 +269,37 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
                     .start_timer();
 
                 // Process chunk request
-                let result_label = match self.process_chunk_request(peer.clone(), *request.clone())
-                {
-                    Ok(()) => counters::SUCCESS_LABEL,
-                    Err(error) => {
+                self.process_chunk_request(peer.clone(), *request.clone())
```

Comment:
is this actually more accessible than

```
let res = ...;
if let Err(error) = res { 
...
] else {
}

res
```

Reply:
The approach I've taken saves a few lines and I don't think it's super unclear, but I'll adopt your approach because it's more explicit! 😄